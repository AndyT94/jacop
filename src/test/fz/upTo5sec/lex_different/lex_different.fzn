array [1..4] of var int: vector1:: output_array([1..4]) = [5,2,7,1];
array [1..4] of var int: vector2:: output_array([1..4]) = [5,3,7,1];
solve  satisfy;
