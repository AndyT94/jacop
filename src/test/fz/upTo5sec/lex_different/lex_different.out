vector1 = array1d(1..4, [5, 2, 7, 1]);
vector2 = array1d(1..4, [5, 3, 7, 1]);
----------